src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
<img width="970" alt="screen shot 2017-08-07 at 9 00 50 pm" src="https://user-images.githubusercontent.com/56682/29051842-8c08a74c-7bb3-11e7-89ce-81b5fd9202a4.png">

That doesn't look right?